src/main/java/com/skilld/rundeck/plugin/step/kubernetes/KubernetesStep.java

```diff
@@ -144,6 +148,24 @@ public void executeStep(PluginStepContext context, Map<String,Object> configurat
 			if(null != configuration.get("activeDeadlineSeconds")){
 				jobConfiguration.setActiveDeadlineSeconds(Long.valueOf(configuration.get("activeDeadlineSeconds").toString()));
 			}
+			if(null != configuration.get(PERSISTENTVOLUME)) {
```

Comment:
This is great (to use the constant in the configuration.get call); would you mind changing the other calls to configuration.get so we would have it clean everywhere then ?